docs/ibm-cp4na-log-message.md

```diff
@@ -16,13 +16,40 @@ The module will accept the following parameters:
 | --- | --- | --- | --- | 
 | message_direction | str | the `message_direction` to be included in the logs | Yes |
 | message_type | str | the `message_type` to be included in the logs | Yes |
-| message_data | str | the `message_data` to be included in the logs | No |
+| message_data | str | the `message` to be included in the logs | No |
 | content_type | str | the `content_type` to be included in the logs | Yes if `message_data` is provided |
 | protocol | str | the `protocol` to be included in the logs | Yes |
 | protocol_metadata | dict | the `protocol_metadata` to be included in the logs (will be formatted as a JSON string by the module/driver) | No |
 | external_request_id | String | the `tracectx.externalrequestid` to be included in the logs | Yes | 
 
-**Note**: supported values for `message_direction` is : `sent` or `received`
+**Notes**: 
+  1. Supported values for `message_direction` are : `sent` and `received`.
+  2. Supported values for `message_type` are : `request`, `response` and `message`.
+  3. Supported values for `protocol` are : `http`, `cmd` or any other protocols.
+  4. If `protocol` is `http`, `protocol_metadata` can have following properties:
+ 
+     ```
+     For message_type request: 
+       uri - The URL of the request
+       headers - Name/value pair of any relevant headers included in the request (should not include headers with sensitive data, such as `Authorization`)
+       method - HTTP method of the request
+     For message_type response:
+       status_code - HTTP status code
+       headers - Name/value pair of any relevant headers included in the response (should not include headers with sensitive data, such as `Authorization`)
+     ```
+     User can also add additional properties under `protocol_metadata` if required. It will vary depending on the ansible module (E.g: ansible.builtin.uri) which will be used to invoke API calls.
+
+  5. If `protocol` is `cmd`, `protocol_metadata` can have following properties:
+
+     ```
+     For message_type request: 
+       command - The command used for executing the request
+
+     For message_type response:
+       exit_code - Exit code of the command
+     ```
+  6. If `message_type` is message, then `protocol_metadata` values will be decided by users.
```

Comment:
Not sure I understand this point? Isn't `protocol_metadata` always decided by the users, say for example they use a `protocol` that isn't `http` or `cmd`? So this is the same as point 7 below.

Reply:
Thanks for pointing it out. Yes, I have removed point 6 and made point 7 as point 6 now with some little modifications.